src/ontology/uberon-edit.obo

```diff
@@ -224513,6 +224513,18 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8900000
+name: sensory corpuscle
+def: "A mechanosensory complex in the terminal end of some nerves that forms a corpuscle. It consists of one or more neuronal terminations and associated non-neuronal cells." [PMID:35356056]
+synonym: "COEC" EXACT [PMID:35356056]
```

Comment:
In Uberon, acronyms are related synonyms (in case it refers a different term as well) and with the annotation property has_synonym_type: abbreviation (as an entity IRI).